src/expac.c

```diff
@@ -43,7 +43,7 @@
 #define DEFAULT_DELIM        "\n"
 #define DEFAULT_LISTDELIM    "  "
 #define DEFAULT_TIMEFMT      "%c"
-#define SIZE_TOKENS          "BKMGTPEZY\0"
+#define SIZE_TOKENS          "BKMGTPEZYRQ"
```

Comment:
It's a pretty far reach that we need to care about even exabytes here. I don't think we need to extend to ronnabytes or quettabytes here...

Reply:
Don't let the electron developers hear you, or they might take it as a challenge

Of course I can change this, but does it do any harm? It's generally good practice to prepare for extremes even if they seem unlikely

Reply:
I'm a software engineer by day, and while I understand your advice, this is also well within the realm of YAGNI.

Let me know when even the first exabyte hard disks (or even tape reels) make it to market.